src/program/services/scrapers/shared.py

```diff
@@ -140,6 +153,7 @@ def _parse_results(item: MediaItem, results: Dict[str, str], log_msg: bool = Tru
             torrents_dict[torrent.infohash] = Stream(torrent)
         logger.log("SCRAPER", f"Kept {len(torrents_dict)} streams for {item.log_string} after processing bucket limit")
         return torrents_dict
+    logger.debug(f"No valid torrent remains after filtering for {item.log_string}: {raw_title}")
```

Comment:
We're given a list of streams to parse here in this function, so this line would only read out the last stream that gets parsed from the list. This line can be removed as we handle individual stream logging throughout this function already.

Reply:
Just a additional explicit debug checkpoint never hurt, updated to remove ```raw_title```